README.md

```diff
@@ -52,7 +52,8 @@ This third approach is taken by Django to facilitate [swapping the auth.User mod
 Swapper is essentially a simple API wrapper around this existing functionality.  Note that Swapper is primarily a tool for library authors; users of your reusable app generally should not need to know about Swapper in order to use it.  (See the notes on [End User Documentation](#end-user-documentation) below.)
 
 ### Real-World Example
-Swapper is used extensively in the [vera] extension to [wq.db].  vera provides [7 inter-related models], each of which can be swapped out for custom implementations.  (Swapper actually started out as part of [wq.db.patterns], but was extracted for more general-purpose use.)
+
+Swapper is used extensively in several OpenWISP projects to facilitate customization.  For example, the [openwisp-users] package leverages `swapper` to allow users to add additional fields and functionality to the base `Organization`, `OrganizationUser`, `OrganizationOwner`, and `Group` models.
```

Comment:
The most notable OpenWISP modules are:

- https://github.com/openwisp/openwisp-controller
- https://github.com/openwisp/openwisp-radius

It would be great if we could mention them! I wouldn't mention each model because the list would be come very long :joy:

Reply:
I updated the section and also cleaned up some old Django references.